schemas.py

```diff
@@ -0,0 +1,39 @@
+from typing import Optional
+from datetime import date
+
+from pydantic import BaseModel
+
+
+class BookBase(BaseModel):
+    title: str
+    summary: Optional[str] = None
+    publication_date: date

```

Comment:
The `publication_date` field in `BookBase` is required. Ensure that this aligns with your application's requirements, as it will require a publication date to be provided whenever a book is created.